scripts/apps/search/controllers/MultiActionBarController.js

```diff
@@ -116,6 +116,16 @@ export function MultiActionBarController(
         multi.reset();
     };
 
+    this.canEditMetadata = function() {
```

Comment:
Consider using `Array.every` and `Array.includes`, it looks more semantic and clean.

```
this.canEditMetadata = () => multi.getItems()
    .every(() => !item.lock_user && ['picture', 'video', 'audio'].includes(item.type));
```